### PR TITLE
Tvm explorer infrastructure

### PR DESCRIPTION
**Pull request by chunit-quic**
Hi community:

Follow the guide in the README.md, this PR is summited for the forum topic:
https://discuss.tvm.apache.org/t/tvm-explorer-infrastructure/13457
Thank you!

### CODE REVIEW

**Review by tkonolige — APPROVED** [review]
Thanks for doing all this work the improve debugging in TVM! I think this is a great step forward.

File: rfcs/0092-TVM-Explorer-Infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,699 @@
...
+    <img src="./assets/0092/callstack_of_schedule_propagation.png?" width="500" height="500">
+  </p>
+  
+  Finally, a series of APIs will be created accordingly for user to access the `Schedule` instance from 
+Relay build module.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* Process extra debug information would cause longer compilation time:
```
[the remark this answers is not in the document]
Comment (chunit-quic) [reply rfcs/0092-TVM-Explorer-Infrastructure.md:620]: Hi @tkonolige, 
Thanks for comments! :D
Yes, we have a profiling results in the forum.
For your reference. You can find the details in the end of this post.
https://discuss.tvm.apache.org/t/pre-rfc-tvm-explorer-infrastructure/13457/14?u=chunit

**Review by areusch — APPROVED** [review]

**Comment by areusch** [comment]
i think we are good on the pre-RFC thread, let's merge this then. @chunit-quic can you create an RFC tracking issue (i.e. "feature tracking" [here](https://github.com/apache/tvm/issues/new/choose)) for this one?

**Comment by chunit-quic** [comment]
> i think we are good on the pre-RFC thread, let's merge this then. @chunit-quic can you create an RFC tracking issue (i.e. "feature tracking" [here](https://github.com/apache/tvm/issues/new/choose)) for this one?

Hi @areusch , 
Thanks a lot for your help! :D
I have created the tracking issue and here is the [link](https://github.com/apache/tvm/issues/13116). Thank you. :)

**Pull request merged**